docs/release-notes/release-notes-0.8.3.md

```diff
@@ -0,0 +1,253 @@
+To avoid this possible violation of Joinmarket's privacy model, after this change, Joinmarket will *ONLY* source commitments from the same mixdepth as the inputs to the Coinjoin. Takers should be aware that this may lead to more cases in which they run out of possible commitments.
+
+`40768cf` Change taker's commitments logic
+
+### Change to mixdepth selection algorithm in yg-privacyenhanced.py
+
+See [PR 808](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/808). This represents a probably somewhat significant improvement in the quality of the choice of mixdepth made by the `yg-privacyenhanced` yieldgenerator (which should be considered default; the `yield-generator-basic.py` is essentially legacy at this point). In short, it makes it a priority to concentrate coins into one mixdepth (or a clump of mixdepths) where possible, and removes an inherent bias towards picking a lower mixdepth that previously existed (technically there is still a *very* small bias but it's not relevant).
+
+A reminder that maker-users of the codebase can also take a look at [this](https://github.com/JoinMarket-Org/custom-scripts/blob/master/yield-generators/yg-acyclic.py).
+
+`993003e` Change mixdepth selection of yg-privacyenhanced
+
+
+### Logging changes
+
+Some fixes/improvements in logging. We send log output to STDOUT instead of STDERR for those who find that useful; we disable colored output in case output is piped to a file (but see one point still unfinished [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/911)); we make sure invalid IRC messages show up in full in the log and also there is a slight improvement in the formatting of transaction feerates.
+
+`96c03ed` Disable coloured output if stdout is not a terminal
+
+`f2362dc` Log whole IRC message on "bad command"
+
+`60622ff` Change log output to STDOUT from STDERR
+
+`a982ea0` Better tx feerate formatting
+
+### Bugfix to `max_sweep_fee_change` usage
+
+For background, you may want to read [this](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.8.1.md#constraining-sweep-transaction-fees) from the 0.8.1 release notes. The above mentioned config setting is currently at 0.8 by default but there was a bug in the implementation such that the calculated ratio was the reciprocal of what was intended; this error is fixed here.
```

Comment:
~above mentioned~ above-mentioned

_The adjective “above-mentioned” is spelled with a hyphen._

~default but~ default, but

_Use a comma before ‘but’ if it connects two independent clauses (unless they are closely connected and short)._

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/912/commits/f1e440cd596e5d46d9da9d6bea0db9e1489d988d